changelog.md

```diff
@@ -1,3 +1,7 @@
+### v10.8.11
```

Comment:
I guess this change may break TS so would just consider making it 10.9.0

Reply:
ok